lighthouse-core/audits/bootup-time.js

```diff
@@ -72,19 +72,55 @@ class BootupTime extends Audit {
     return result;
   }
 
+  /**
+   * Summarize JS costs by URL
+   * @param {Map<string, Object<string, number>>} executionTimings
+   * @param {LH.Audit.Context} context
+   * @return {{results: Object, totalBootupTime: number, extendedInfo: Object}}
```

Comment:
`Object` is just `any`, so these should be updated. They have types on master (though as local variables), so you can just use those